app/models/favourite_room.rb

```diff
@@ -0,0 +1,4 @@
+class FavouriteRoom < ApplicationRecord
+  belongs_to :room
+  belongs_to :user
+end
```

Comment:
why the indirection over FavouriteRoom? Wouldn't it be possible to simply store references to users? The FavouriteRoom model leads imo to a confusing relationship because a room has now many favourite rooms which sounds weird.

Reply:
Ah, probably because this model is required by the `has many through` relationship, right? Fine with me since it's the recommended way for extensibility of the relationship (the alternative would be the `has_and_belongs_to_many` we used in the other models)

Reply:
This indirection may come in helpful if we wanted to extend favourites to buildings, people & Co, right?